{"@t":"2020-03-21T23:03:01.4578583Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":59,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:08:05.4959469Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:13:09.5291358Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":75,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:18:13.5515699Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:23:17.5924729Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:28:21.6165103Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:33:25.6323474Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:38:29.6581071Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:43:33.7043653Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:48:37.7232253Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-21T23:53:41.7785803Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T00:22:00.0751071Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T00:27:04.0962066Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T00:32:08.1082986Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":75,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:05:01.0381817Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:10:05.0718421Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:15:09.0937264Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":75,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:20:13.1197652Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:25:17.1529845Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":59,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:30:21.1727505Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:35:25.1886661Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:40:29.2107370Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T14:45:33.2367527Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":21,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:24:23.9061980Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:29:27.9322075Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:34:31.9572695Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:39:35.9722060Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:44:39.9818210Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:49:44.0585634Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:54:48.0886808Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T15:59:52.1131429Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:04:56.1450808Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:10:00.1925991Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:15:04.2604267Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":72,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:20:08.3344889Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":50,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:25:12.3909779Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:30:16.4416530Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:35:20.5044786Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:40:24.5642517Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:45:28.6283485Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:50:32.6528956Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T16:55:36.6790301Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:00:40.7027396Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:05:44.7277328Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:10:48.7503244Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:15:52.7758460Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:20:56.7931939Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:26:00.8152227Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:31:04.8293482Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:36:08.8479245Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:41:12.8640006Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:46:16.8740968Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:51:20.8912180Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T17:56:24.9103430Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:01:28.9314532Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:06:32.9557834Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:13:20.8153319Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:18:24.8276990Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:23:28.8597340Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:28:32.8758495Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:33:36.8909350Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:38:40.9160015Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:43:44.9370220Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:48:48.9534134Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:53:52.9815503Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T18:58:57.0006088Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:04:01.0142876Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:09:05.0352080Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:14:09.0628683Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:19:13.0854824Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:24:17.1082280Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:29:21.4080406Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:34:25.4363456Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:39:29.4530806Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:44:33.5146502Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:49:37.6103013Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:54:41.6652892Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T19:59:45.7213638Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:04:49.7805325Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":70,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:09:53.8123817Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:14:57.8291364Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":52,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:20:01.8675976Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:25:06.0431062Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":52,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:30:10.4070256Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:35:14.4733932Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:40:18.4847621Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:45:22.5041583Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":52,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:50:26.5320256Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":52,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T20:55:30.5670917Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:00:34.6329784Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:05:38.6723665Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:10:42.7103173Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:15:47.5666587Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:20:51.5843929Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:25:55.6240182Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":52,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:30:59.6513116Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:36:03.6856712Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:41:07.7177042Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:46:11.7445614Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:51:15.7680560Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T21:56:19.7914974Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":63,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:01:23.8189786Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":73,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:06:27.8473939Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:11:31.8773263Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:16:35.9124331Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:21:39.9411350Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":75,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:26:43.9727951Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:31:48.0019672Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:36:52.0306373Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:41:56.0533118Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":26,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:47:00.0745952Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:52:04.0995433Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":75,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-22T22:57:08.1276742Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt nægtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":30568,"ProcessName":"iisexpress","ThreadId":36,"AppDomainId":8,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
